packages/fetchai/skills/tac_control_contract/behaviours.py

```diff
@@ -222,7 +222,7 @@ def _request_mint_items_transaction(self, game: Game) -> None:
             kwargs=ContractApiMessage.Kwargs(
                 {
                     "deployer_address": self.context.agent_address,
-                    "recipient_address": self.context.agent_address,
+                    "recipient_address": agent_state.agent_address,
```

Comment:
This was one key problem...took me a while to find!

Reply:
Of course the deployer and recipient shouldn't be the same.